### PR TITLE
Adding new vendor IncrementX 

### PR DESCRIPTION
**Pull request by Prebid-Vertoz**

### CODE REVIEW

File: 3p/integration.js [hunk 1]
```diff
@@ -430,6 +431,7 @@ register('imedia', imedia);
 register('imobile', imobile);
 register('imonomy', imonomy);
 register('improvedigital', improvedigital);
+register('incrementx', incrementx);
```
Comment (powerivq) [inline 3p/integration.js:434]: This file does not need to be changed any more. Could you revert it?

**Review by powerivq — APPROVED** [review]

File: 3p/integration.js (hunk 1 above)
Comment (Prebid-Vertoz) [reply 3p/integration.js]: we have removed the lines

**Comment by powerivq** [comment]
Still need to fix the check failures. Plz look at the log. Should be pretty easy fix.

**Comment by Prebid-Vertoz** [comment]
> Still need to fix the check failures. Plz look at the log. Should be pretty easy fix.

We have removed the lines 3p/integration.js and ads.out can you please check and let us know if the commits are correct ?

**Comment by powerivq** [comment]
@Prebid-Vertoz It is still failing here: https://app.circleci.com/pipelines/github/ampproject/amphtml/28467/workflows/b08c51be-f7ff-4896-88c5-0e38a5ca5cc8/jobs/604813

**Comment by powerivq** [comment]
Also you need to sign the CLA. You can follow the button in the comment above,.

**Comment by Prebid-Vertoz** [comment]
> @Prebid-Vertoz It is still failing here: https://app.circleci.com/pipelines/github/ampproject/amphtml/28467/workflows/b08c51be-f7ff-4896-88c5-0e38a5ca5cc8/jobs/604813
> 
> Hint: You need to remove the trailing whitespace.

Remove the whitespaces and also sign the CLA

**Comment by powerivq** [comment]
Please fix: https://app.circleci.com/pipelines/github/ampproject/amphtml/28498/workflows/2241aba2-209b-44f3-bb5f-341351f90153/jobs/605622

You can run `amp lint` locally to ensure you fix all lint issues.

**Review by powerivq — APPROVED** [review]

**Comment by powerivq** [comment]
Added @banaag for validator approval.

**Comment by Prebid-Vertoz** [comment]
> Added @banaag for validator approval.

Hello,
Any update?

**Review by erwinmombay — APPROVED** [review]

**Pull request merged**